src/components/assets/Assets.vue

```diff
@@ -9,34 +9,49 @@
             :handle-update-token-balances="handleUpdateTokenBalances"
           />
         </div>
-        <div v-else><NativeAssetList /></div>
+        <div v-else class="container--assets">
+          <NativeAssetList />
+          <XcmNativeAssetList v-if="isEnableXcm" />
+        </div>
       </div>
     </div>
   </div>
 </template>
 <script lang="ts">
 import Account from 'src/components/assets/Account.vue';
 import NativeAssetList from 'src/components/assets/NativeAssetList.vue';
+import XcmNativeAssetList from 'src/components/assets/XcmNativeAssetList.vue';
 import EvmAssetList from 'src/components/assets/EvmAssetList.vue';
 
 import { useStore } from 'src/store';
 import { defineComponent, computed, ref, watchEffect } from 'vue';
 import { useCbridgeV2 } from 'src/hooks';
 import { LOCAL_STORAGE } from 'src/config/localStorage';
 import { wait } from 'src/hooks/helper/common';
+import { endpointKey, getProviderIndex } from 'src/config/chainEndpoints';
 
 export default defineComponent({
   components: {
     Account,
     NativeAssetList,
     EvmAssetList,
+    XcmNativeAssetList,
   },
   setup() {
     const { tokens, ttlErc20Amount, handleUpdateTokenBalances } = useCbridgeV2();
     const isDisplay = ref<boolean>(false);
     const store = useStore();
     const selectedAddress = computed(() => store.getters['general/selectedAddress']);
     const isH160 = computed(() => store.getters['general/isH160Formatted']);
+    const currentNetworkIdx = computed(() => {
+      const chainInfo = store.getters['general/chainInfo'];
+      const chain = chainInfo ? chainInfo.chain : '';
+      return getProviderIndex(chain);
+    });
+
+    const isEnableXcm = computed(
+      () => !isH160.value && currentNetworkIdx.value !== endpointKey.SHIBUYA
```

Comment:
This is a condition for[ native (SS58) assets component](https://github.com/AstarNetwork/astar-apps/pull/397/files/c1738e5f976f885442d98b0c5503e3ff4d6fe00f#diff-49ecdb11a4f48eeb237de82dc701b8ff7d3ce0bd904070f239c172efb2f5cf34R14).

We can set the different condition for EVM mode [here](https://github.com/AstarNetwork/astar-apps/pull/397/files/c1738e5f976f885442d98b0c5503e3ff4d6fe00f#diff-49ecdb11a4f48eeb237de82dc701b8ff7d3ce0bd904070f239c172efb2f5cf34R14).

Reply:
@sirius651 
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/397/commits/36b0615558f7575fe8b68b55c91956959db104e8) commit since we already have [this](https://github.com/AstarNetwork/astar-apps/blob/c1738e5f976f885442d98b0c5503e3ff4d6fe00f/src/components/assets/Assets.vue#L6) line.